fem/linearform.hpp

```diff
@@ -78,11 +82,23 @@ class LinearForm : public Vector
    /// Copy construction is not supported; body is undefined.
    LinearForm(const LinearForm &);
 
+   /// Defaults values to be used in constructors
+   void Init()
+   {
+      ext = nullptr;
+      extern_lfs = 0;
+      fast_assembly = false;
```

Comment:
Maybe just provide these as the default values for the data members and then we can delete `LinearForm::Init` entirely?

Reply:
Fixed in https://github.com/mfem/mfem/pull/3235/commits/343e7ca7c8c5f4b432f97f8632702f8ce5ef2c6c